terraform/modules/storage/object/minio/outputs.tf

```diff
@@ -7,4 +7,9 @@ output "generated_env_vars" {
     "S3__Password"              = var.password
     "S3__MustForcePathStyle"    = true
   })
-}
\ No newline at end of file
+}
+
+output "volumes" {
+  description = "Volumes that agents must mount to access the object storage"
```

Comment:
not only agents

Reply:
Arguably, the submitter is also an agent. What naming would you suggest?

Reply:
agents and submitters